### PR TITLE
adds jshint CI

### LINKED ISSUE #139: upgrade bowling challenge CI
ensure jshint working - details here: https://houndci.com/configuration

### PR DESCRIPTION
**Pull request by tansaku**
fixes #139

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    // Enforcing
+    "bitwise"       : true,     // true: Prohibit bitwise operators (&, |, ^, etc.)
+    "camelcase"     : false,    // true: Identifiers must be in camelCase
+    "curly"         : true,     // true: Require {} for every new block or scope
+    "eqeqeq"        : true,     // true: Require triple equals (===) for comparison
+    "forin"         : true,     // true: Require filtering for..in loops with obj.hasOwnProperty()
+    "freeze"        : true,     // true: prohibits overwriting prototypes of native objects such as Array, Date etc.
+    "immed"         : false,    // true: Require immediate invocations to be wrapped in parens e.g. `(function () { } ());`
+    "indent"        : 4,        // {int} Number of spaces to use for indentation
```
Comment (danldb) [inline .jshintrc]: shouldn't this be 2?

Comment (tansaku) [inline .jshintrc]: not sure

**Comment by tansaku** [comment]
@danldb @spike01 will you be happy with this if I turn off the semi-colon alerts?  have a look at the other open pull requests to see examples

File: .jshintrc (hunk 1 above)
Comment (spike01) [inline .jshintrc]: It should be 2

**Pull request merged**